browser/ui/views/brave_default_browser_dialog_view.cc

```diff
@@ -44,27 +47,71 @@ namespace {
 
 constexpr int kPadding = 24;
 
-class NoSnappedBubbleFrameView : public views::BubbleFrameView {
+class DontAskAgainButton : public views::LabelButton {
  public:
-  using BubbleFrameView::BubbleFrameView;
-  ~NoSnappedBubbleFrameView() override = default;
+  METADATA_HEADER(DontAskAgainButton);
 
-  NoSnappedBubbleFrameView(const NoSnappedBubbleFrameView&) = delete;
-  NoSnappedBubbleFrameView& operator=(const NoSnappedBubbleFrameView&) = delete;
+  explicit DontAskAgainButton(PressedCallback callback)
+      : LabelButton(std::move(callback)) {
+    SetFontList();
+    SetText(brave_l10n::GetLocalizedResourceUTF16String(
+        IDS_BRAVE_DEFAULT_BROWSER_DIALOG_DONT_ASK));
+  }
+  DontAskAgainButton(const DontAskAgainButton&) = delete;
+  DontAskAgainButton& operator=(const DontAskAgainButton&) = delete;
+
+ private:
+  void SetFontList() {
+    gfx::FontList font_list;
+    constexpr int kFontSize = 13;
+    font_list.Derive(kFontSize - font_list.GetFontSize(),
+                     font_list.GetFontStyle(), gfx::Font::Weight::NORMAL);
+    label()->SetFontList(font_list);
+  }
+
+  // views::LabelButton overrides:
+  void AddedToWidget() override {
```

Comment:
nit: I think OnThemeChanged() is proper for this work in a semantic point of view.